enmasse/README.adoc

```diff
@@ -20,6 +20,8 @@ The deployment is pretty straightforward. I used what was the current release at
 Once you have cloned the repository you can follow the http://enmasse.io/documentation/0.19.0/[instructions] to deploy EnMasse. In short the following will have the user admin deploying the EnMasse components in the perftest project using provided ansible playbook:
 [source,bash]
 ----
+$ oc new-project perftest
+create a persitence volume as statefulset requires it,one can use the template provided here(oc create -f brokerpv.yaml )
```

Comment:
Yes you are right we should state that a 2GB PV is required for the provisioning of EnMasse. That said we don't have brokerpv.yaml in the repo and the file would be different for every OCP cluster. We could just have before "In short the following will ..." something like: "Note that you need to have a 2 GB persistent volume available"